package.json

```diff
@@ -45,7 +45,7 @@
         "url": "https://github.com/box/box-ui-elements/issues"
     },
     "engines": {
-        "node": ">=18.x",
+        "node": ">=18.x || ^22.0",
```

Comment:
you can remove this change since `>=` already allows versions greater than 18 to work. that's how 20.x and 22.x already work for BUIE

Reply:
I feel that leaving in 22 makes it more explicit that we're supporting 22, although there is no net change. 

when i first went to look at this repo, it wasn't clear to me that we were supporting node 20, for example. the only place that mentioned node 20 was developing.md and the circleCI config.

Reply:
ok, updated